docs/reference/ml/df-analytics/apis/delete-trained-model.asciidoc

```diff
@@ -0,0 +1,59 @@
+[role="xpack"]
+[testenv="basic"]
+[[delete-inference]]
+=== Delete trained model API
+[subs="attributes"]
+++++
+<titleabbrev>Delete trained model</titleabbrev>
+++++
+
+Deletes an existing trained {infer} model that is currently not referenced by an 
+ingest pipeline.
```

Comment:
> that is not currently referenced by an ingest pipeline

This seems to indicate that a failure occurs if you try to call this when the model is currently referenced.  It makes me wonder what sort of failure you'd get.  If we know, I think that info should be added a response codes section in this API